buildSrc/src/main/java/com/google/firebase/gradle/plugins/LibraryGroupRegistrar.kt

```diff
@@ -25,11 +25,15 @@ class LibraryGroupRegistrar {
     val maxVersion =
       librariesByGroup
         .get(libraryGroup)
```

Comment:
The more Kotlin iodimatic way would be:
```kotlin
librariesByGroup[libraryGroup]
```

But I'm not against this since it's a sequence of method calls